Browser/entry.py

```
@@ -206,11 +206,11 @@ def show_versions():
     version_file = CURRENT_FOLDER / "version.py"
     version_text = version_file.read_text()
     match = re.search(r"\"\d+\.\d+.\d+\"", version_text)
-    browser_lib_version = match.group(0)
+    browser_lib_version = match.group(0) if match else "unknown"
```

I would just raise an exception, it gets noticed better in that cas